src/test/java/org/tugraz/sysds/test/functions/federated/FederatedUrlParserTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2019 Graz University of Technology
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.tugraz.sysds.test.functions.federated;
+
+import org.tugraz.sysds.runtime.instructions.fed.InitFEDInstruction;
+import org.tugraz.sysds.runtime.DMLRuntimeException;
+import org.tugraz.sysds.conf.DMLConfig;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class FederatedUrlParserTest {
+
+    @Test(expected = DMLRuntimeException.class)
```

Comment:
yes, I'm nitpicking, but maybe there's a better exception to throw ;-)

Reply:
changed to IllegalArgumentException